static/compiler.js

```diff
@@ -504,13 +504,12 @@ define(function (require) {
     };
 
     Compiler.prototype.onCompilerSetDecorations = function (id, lineNums) {
-        if (id === this.id) {
+        if (id == this.id) {
```

Comment:
I can confirm. It seems like in  '_.each(array, function(object, index) {...} ' index is a string. What should I do? Leave it like it is, or cast it to int before sending it to the event hub?